man/veritytab.xml

```diff
@@ -186,8 +186,8 @@ This is based on crypttab(5).
         <term><option>fec-device=<replaceable>PATH</replaceable></option></term>
 
         <listitem><para>Use forward error correction (<acronym>FEC</acronym>) to recover from corruption if hash verification fails. Use
-        encoding data from the specified device. The fec device argument can be block device or file image. For format,
-        if fec device path doesn't exist, it will be created as file. Note: block sizes for data and hash devices must
+        encoding data from the specified device. The fec device argument can be block device or file image.
+        If fec device path doesn't exist, it will be created as file. Note: block sizes for data and hash devices must
```

Comment:
Arguably that whole sentence should be removed, since currently veritytab is not used by anything that creates verity data. Most of these descriptions were copied from veritysetup(1) where "format" is one of the commands that actually creates this data.

The `hash=` entry also pointlessly mentions the format command. The really weird one is `uuid=`, which in this fifth field is defined solely in what the corresponding parameter does when passed to `veritysetup format`, but in systemd it is really used only when `superblock=false` and a custom GUID was used when generating, to ensure verification works properly.

Reply:
I'm not sure of this part.Is systemd-veritysetup@.service that reads veritytab then using veritysetup?

Reply:
[systemd-veritysetup-generator(8)](https://www.freedesktop.org/software/systemd/man/latest/systemd-veritysetup-generator.html#) creates systemd-veritysetup@.service files based on veritytab.

Those service files call systemd-veritysetup. systemd-veritysetup is basically just a cut down version of the [original veritysetup](https://man7.org/linux/man-pages/man8/veritysetup.8.html) program (part of [cryptsetup](https://gitlab.com/cryptsetup/cryptsetup)) which only supports the equivalent of the original programs "open" and "close" commands. The original programs supports several additional commands.

However the documentation for all the veritytab options were borrowed (read copy-pasted) from the manpage of the original veritysetup tool, and thus sometimes talk about other commands from that original program like "format", which are not relevant to veritytab.

Reply:
`and it does not appear that systemd creates that path in any scenario, hence why I suggest droping the whole sentence in question`
`crypt_format`  is used to do the real work in systemd-veritysetup which is defined in cryptsetup. So I think the logic seems right, but something wrong with the sentence.